pulp_container/tests/functional/api/test_build_images.py

```diff
@@ -29,15 +30,59 @@ def containerfile_name():
         yield containerfile.name
 
 
-def test_build_image(
+@pytest.fixture
+def create_file_and_container_repos_with_sample_data(
+    container_repository_api,
+    file_bindings,
+    file_repository_factory,
+    gen_object_with_cleanup,
+    tmp_path_factory,
+):
+    container_repo = gen_object_with_cleanup(
+        container_repository_api, ContainerContainerRepository(**gen_repo())
+    )
+
+    filename = tmp_path_factory.mktemp("fixtures") / "example.txt"
+    filename.write_bytes(b"test content")
+    file_repo = file_repository_factory(autopublish=True)
+    upload_task = file_bindings.ContentFilesApi.create(
+        relative_path="foo/bar/example.txt", file=filename, repository=file_repo.pulp_href
+    ).task
+    monitor_task(upload_task)
+
+    return container_repo, file_repo
+
+
+@pytest.fixture
+def build_image(container_repository_api):
+    def _build_image(repository, containerfile, artifacts=None, repo_version=None):
+        # workaround for drf-spectacular not allowing artifacts=None and raising an exception
+        # (a bytes-like object is required, not 'NoneType')
```

Comment:
Does it mean that when I want to build a new image from a Containerfile that does not require any additional artifacts, I cannot use the python bindings?

Reply:
I think so. Testing (from main branch) with:
```
    build_response = container_repository_api.build_image(
        repository.pulp_href, containerfile=containerfile_name, artifacts=None
    )
```

will fail:
```
E               TypeError: a bytes-like object is required, not 'NoneType'
```

Reply:
This is so unfortunate. Is there any chance that we provide an invalid openapi schema for the generation?